### PR TITLE
issues: add templates for "New Issue" on github

### PR DESCRIPTION
**Pull request by alltilla**
1. Bug report
2. Documentation issue
3. Feature request

Example: https://github.com/tensorflow/tensorflow/issues/new/choose

### CODE REVIEW

**Comment by Kokan** [comment]
Please additionally remove the *old* issue template.

**Comment by kira-syslogng** [comment]
Build SUCCESS

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -1,3 +1,12 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
```
Comment (MrAnno) [inline .github/ISSUE_TEMPLATE/bug_report.md]: >  (Please, remove the unnecessary notes, including this one.)

I've seen projects using HTML comments to give instructions. This way we wouldn't have to ask them to remove anything.

For example:

```

## Steps to reproduce

<!-- A description of how to trigger this bug. -->
```

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: 'enhancement'
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
```
Comment (MrAnno) [inline .github/ISSUE_TEMPLATE/feature_request.md]: The mentioned pattern could be used here as well, for example:

```
## Description of the problem

<!--
Is your feature request related to a problem? Please describe.
A clear and concise description of what the problem is. Ex. I'm always frustrated when
-->

## Description of the feature

<!--  A clear and concise description of what you want to happen.  -->

## Alternatives

## Additional context

<!-- Add technical implementation details, screenshots, any other context. -->
```

**Comment by MrAnno** [comment]
:+1: 
This will render a page like this:
https://github.com/tensorflow/tensorflow/issues/new/choose

What do you think about adding a "Question" category as well?

**Comment by kira-syslogng** [comment]
Build SUCCESS

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 2 above)
Comment (alltilla) [reply .github/ISSUE_TEMPLATE/feature_request.md]: Thanks, done! :)

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (alltilla) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Nice idea, done! Thanks :)

**Comment by kira-syslogng** [comment]
Build SUCCESS

**Comment by alltilla** [comment]
@fekete-robert: As we talked about it, I added a template for "Documentation issues". Could you check if it is good for you? :)

**Comment by fekete-robert** [comment]
Looks great, thank you!

**Comment by kira-syslogng** [comment]
Build SUCCESS

**Comment by kira-syslogng** [comment]
Build SUCCESS

**Pull request merged**